cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -413,8 +424,10 @@ public void execute() {
                 .addTask(new CreateBaloTask(), isSingleFileBuild)   // create the BALOs for modules (projects only)
                 .addTask(new CreateBirTask())   // create the bir
                 .addTask(new CopyNativeLibTask())    // copy the native libs(projects only)
+                .addTask(new CopyChoreoExtensionTask(), isChoreoExtensionSkipped)   // copy the Choreo extension
                 .addTask(new CreateJarTask(skipCopyLibsFromDist))   // create the jar
                 .addTask(new CopyResourcesTask(), isSingleFileBuild)
+                .addTask(new CopyObservabilitySymbolsTask(), isSingleFileBuild)
```

Comment:
There are cases like running unit tests, where we skip copying libs from distribution using 'skipCopyLibsFromDist' flag. You might need to use 'skipCopyLibsFromDist' in CopyChoreoExtensionTask. Lets see  if unit tests failed.

Reply:
I think in all our test cases, this will be skipped. Because none of them has the build time flag or the `Ballerina.toml` table. (Except maybe in any test case which uses `ballerina new`). Will confirm once the other `lang-server` related test case issue had been fixed.

Reply:
Resolving as all the integrations are passing.